charts/testkube-api/templates/testworkflows/builtin-templates/postman.yaml

```
@@ -0,0 +1,28 @@
+{{- if .Values.global.testWorkflows.createOfficialTemplates }}
+kind: TestWorkflowTemplate
+apiVersion: testworkflows.testkube.io/v1
+metadata:
+  name: official--postman--beta
+  labels:
+    testkube.io/name: Postman
+    testkube.io/wizard: enabled
+  annotations:
+    testkube.io/categories: API
+    testkube.io/description: Run Postman tests
+    testkube.io/icon: postman
+spec:
+  config:
+    version:
+      description: Postman version to use
+      type: string
+      default: 6-alpine
+    run:
+      description: Run command
+      type: string
+      default: "newman run"
+  steps:
+  - name: Run Postman tests
+    run:
+      image: {{`"postman/newman:{{ config.version }}"`}}
```

Shouldn't that rather be:
```suggestion
      image: "{{`postman/newman:{{ config.version }}`}}"
```

---

Besides, doesn't other template need to be escaped too?

---

Or maybe more like:
```suggestion
      image: {{- toYaml `postman/newman:{{ config.version }}` }}
```